src/process/process.rs

```diff
@@ -265,19 +255,10 @@ impl Process {
 		}
 	}
 
-	/// Preemptively checks if the process is still alive.
 	pub fn terminate(&self) -> ProcessResult<()> {
-		#[cfg(target_family = "unix")]
-		{
-			self.send_signal(Signal::SIGTERM)
-		}
-		#[cfg(not(any(target_family = "unix")))]
-		{
-			todo!()
-		}
+		self.send_signal(Signal::SIGTERM)
 	}
 
-	/// Preemptively checks if the process is still alive.
```

Comment:
Can you explain why you removed these comments? Has the functionality changed?

Reply:
I reread the code and decided to re-add these.